functions/go/enable-gcp-services/README.md

```diff
@@ -0,0 +1,120 @@
+# enable-gcp-services
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The `enable-gcp-services` function generates [GCP project service](https://cloud.google.com/config-connector/docs/reference/resource-docs/serviceusage/service)
+resources from a list of services to enable [GCP APIs](https://cloud.google.com/apis) within a specified project. This allows users to succinctly define all
+the services necessary in a single resource and have tighter control over which services are enabled in a specific project.
+
+<!--mdtogo-->
+
+<!--mdtogo:Long-->
+
+## Usage
+
+`enable-gcp-services` function can be used both declaratively and imperatively.
+
+```shell
+kpt fn eval --image gcr.io/kpt-fn/enable-gcp-services:unstable
+```
+
+The `enable-gcp-services` function does the following:
+
+1. Discovers all `ProjectServiceList` custom resources in a given package.
```

Comment:
Nested packages are scanned as well. You should mention it.

Reply:
added